lib/changelog/stats/s3.ex

```diff
@@ -1,17 +1,23 @@
 defmodule Changelog.Stats.S3 do
+  @config [
+      access_key_id: SecretOrEnv.get("AWS_ACCESS_KEY_ID"),
+      secret_access_key: SecretOrEnv.get("AWS_SECRET_ACCESS_KEY"),
+      host: SecretOrEnv.get("AWS_API_HOST")
+    ]
```

Comment:
These Sentry errors make me think that this gets evaluated at compile time, not runtime: https://changelog-media.sentry.io/issues/4375617018/?project=5668962&query=is%3Aunresolved&referrer=issue-stream&stream_index=0

WDYT @jerodsanto?

Reply:
That's right, good call. I'll move them into a function instead and that should fix this.

Reply:
Looking good!